src/Clapi/NamespaceTracker.hs

```diff
@@ -77,15 +79,14 @@ nstProtocol_ = forever $ liftedWaitThen fwd rev
           (uncurry $ throwOutProvider i)
           (const $ do
               (icd, frcd) <- toInboundClientDigest i d
-              unless (frcdNull frcd) $
-                lift $ sendRev $ Right $ ServerData i $ Frcd frcd
+              sendClientRev i frcd
               sendFwd' i $ Icd icd
           )
           (guardNsClientDigest i d)
     fwd (ClientDisconnect i) = handleDisconnect i
     rev (Originator i, od) = case od of
       Ocid d -> registerSubs i d
```

Comment:
Perhaps we should look at why the lower levels of the relay code are sending us empty `Ocid`s in the first place?